android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt

```diff
@@ -325,6 +330,9 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
             getBoolean(paymentMethodOptions, "requestIncrementalAuthorizationSupport")
         val requestedPriority = paymentMethodOptions?.getString("requestedPriority")
         val captureMethod = params.getString("captureMethod")
+        val offlineBehavior = params.getString("offlineBehavior")
+        val offlineModeTransactionLimit = params.getInt("offlineModeTransactionLimit")
```

Comment:
offlineModeTransactionLimit is not an SDK param, it's something the integration keeps track of itself by checking `offlinePaymentAmountsByCurrency`. We can remove the references to it in our code